README.md

```diff
@@ -35,9 +35,35 @@ $client = new Client($conn, new PurePacker());
 // $client = new Client($conn, new PeclPacker());
 
 $space = $client->getSpace('my_space');
+
+// Selecting all data
 $result = $space->select();
 var_dump($result->getData());
 
+// Inserting data
+$insertValues = [
+    'first insert val',
+    'second insert val',
+];
+$space->insert($insertValues);
+
+// Upserting values
+$upsertValues = [
+    1,
```

Comment:
If $insertValues has `1`, then $upsertValues must have `2`, otherwise you upsert examples are incorrect.